qemu/tests/cfg/nested_libguestfs_unittest.cfg

```diff
@@ -10,16 +10,16 @@
     remove_image_image1 =yes
     start_vm = no
     unittest_timeout = 600
-    # TODO: Add support for s390
-    only ppc64le, x86_64
     variants nested_flag:
         - nested_flag_default:
         - nested_flag_on:
+            only ppc64le, x86_64
```

Comment:
Since s390x does not any option to control the nested feature on qemu side, I am thinking about mapping nested_flag_on(nested=1)/nested_flag_off(nested=0), then libguestfs can run at TCG/KVM mode.